specs/data_structures.md

```diff
@@ -330,7 +330,7 @@ For leaf node `node` of [share](#share) data `d`:
 ```C++
 node.n_min = d.namespaceID
 node.n_max = d.namespaceID
-node.v = h(0x00, d.rawData)
+node.v = h(0x00, d.namespaceID, d.rawData)
```

Comment:
So this would only be the case for original data. For parity shares, this would still stay:
`node.v = h(0x00, d.rawData)`
right? @evan-forbes @adlerjohn

Reply:
This would actually not be the case of original data either! (Pushing the fix soon)